docs/source/conf.py

```diff
@@ -20,13 +23,22 @@
 
 # -- Auto notebook index creation --------------------------------------------
 
-static_indir = "examples/_static"
-rst_outdir = "examples/notebook/"
+example_root_url = "https://delvtech.github.io/elf-simulations-examples/"
+r = requests.get(example_root_url, timeout=5)
+# Find all strings in quotes, which should result in all notebook htmls
+files = re.findall(r"\".*?\"", r.content.decode("utf-8"))
```

Comment:
haha this is silly but looks like it works.  I would say let's make it a little bit more resilient with something like:
files = re.findall(r"\".*\.notebook.html?\"", r.content.decode("utf-8"))
or at least 
files = re.findall(r"\".*\.html?\"", r.content.decode("utf-8"))
in case something changes and strings are introduced to the page.